components/aws/sagemaker/tests/integration_tests/.env.example

```diff
@@ -0,0 +1,12 @@
+# If you would like to override the credentials for the container
+# AWS_ACCESS_KEY_ID=
+# AWS_SECRET_ACCESS_KEY=
+# AWS_SESSION_TOKEN=
+
+REGION=us-east-1
```

Comment:
All test will run in this region only ?

Reply:
This file works as an example of a `.env` file that you can use to configure the integration tests when you run them locally. I will write up a README about how to do this, but it's not included in this PR.